example/storybook/src/ui-components/Radio/styled-components/Icon.tsx

```diff
@@ -1,40 +1,45 @@
-import { CircleIcon } from '../../Icons/Icons';
+import { AsForwarder } from '@dank-style/react';
 import { styled } from '../../styled';
 
 export default styled(
-  CircleIcon,
+  AsForwarder,
   {
     'w': '100%',
     'h': '100%',
     'p': 2,
     'justifyContent': 'center',
     'alignItems': 'center',
-    'color': '$primary600',
+    'borderRadius': 999,
```

Comment:
Can we use borderRadius:'$full' @akash3gtm

Reply:
Okay, will update